packages/back-end/src/services/features.ts

```diff
@@ -115,17 +119,26 @@ export function generateFeaturesPayload({
   groupMap,
   prereqStateCache = {},
   safeRolloutMap,
+  holdoutsMap,
 }: {
```

Comment:
Make sure we're calling `refreshSDKPayload` whenever a holdout is started or stopped